.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Was fighting against my formatter this morning trying to match with your settings